cartography/data/jobs/analysis/aws_ec2_keypair_analysis.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "Analysis jobs for EC2 Key Pairs",
+    "statements": [
+        {
+            "__comment__": "Delete the attribute uploaded",
+            "query": "MATCH (k:EC2KeyPair) WHERE EXISTS (k.uploaded) REMOVE k.uploaded return COUNT(*) as TotalCompleted",
+            "iterative": false
+        },
+        {
+            "__comment__": "Delete the attribute duplicate_keyfingerprint",
+            "query": "MATCH (k:EC2KeyPair) WHERE EXISTS (k.duplicate_keyfingerprint) REMOVE k.duplicate_keyfingerprint return COUNT(*) as TotalCompleted",
+            "iterative": false
+        },
+        {
+            "__comment__": "Delete stale relationships between EC2KeyPairs",
+            "query": "MATCH (k1:EC2KeyPair)-[r:MATCHING_FINGERPRINT]-(k2:EC2KeyPair) WHERE r.lastupdated <> {UPDATE_TAG} DELETE (r) return COUNT(*) as TotalCompleted",
+            "iterative": false
+        },
+        {
+            "__comment__": "Set EC2KeyPairs with MD5 hash keyfingerprints (47 characters, instead of 59 characters) as uploaded = True",
+            "query": "MATCH (k:EC2KeyPair) WHERE length(k.keyfingerprint) = 47 SET k.uploaded = True return COUNT(*) as TotalCompleted",
```

Comment:
What do you think about changing the name to `user_uploaded`?

Reply:
Yeah, that sounds better.